elastic/integrations
# [Integrations] Updated the infrastructure category

- Enhancement

## What does this PR do?

This PR tags the infrastructure integrations under a new category called as `infrastructure` .

## Checklist



## How to test this PR locally
- Clone the PR branch to local.
- Run the command to bring up the stack `elastic-package stack up`
- Navigate to Browse integrations page.
- Select the elastic-agent only option on the page.
- In the left category panel click on the infrastructure category.
- This should result in listing all the integration which is tagged under this new category.

## Related issues

- Closes #4469 

CC: @rameshelastic @lalit-satapathy 
## Screenshots
